src/main/kotlin/com/sourcegraph/cody/edit/sessions/FixupSession.kt

```diff
@@ -398,7 +397,14 @@ abstract class FixupSession(
   override fun dispose() {
     if (project.isDisposed) return
     performedActions.forEach { it.dispose() }
-    editor.document.removeDocumentListener(documentListener)
+
+    try {
+      editor.document.removeDocumentListener(documentListener)
+    } catch (e: Throwable) {
```

Comment:
it will not help

Reply:
the error is already logger as internal error by `logger.error()`

Reply:
Yea you are right. We need to look at it tomorrow then.

Reply:
is it possible to check if `documentListener` is valid before `editor.document.removeDocumentListener`? or does the error come from somewhere else?

Reply:
the error is already reported to the IDE within `removeDocumentListener` - we cannot catch it 😕 

I merged this one and I am going to do the release. I will try to fix this issue later.

Reply:
a few moments later: https://github.com/sourcegraph/jetbrains/pull/1621